python/python/ert_gui/simulation/models/iterated_ensemble_smoother.py

```diff
@@ -28,8 +29,9 @@ def _runAndPostProcess(self, run_context):
         self.ert().getEnkfSimulationRunner().runWorkflows( HookRuntime.PRE_SIMULATION )
 
         self.setPhaseName("Running forecast...", indeterminate=False)
-        num_successful_realizations = self.ert().getEnkfSimulationRunner().runSimpleStep(self._job_queue, run_context)
 
+        num_successful_realizations = self.ert().getEnkfSimulationRunner().runSimpleStep(self._job_queue, run_context)
+        num_successful_realizations += arguments['successful_realizations']
```

Comment:
I think the `successful_realizations` argument is a bit ambiguous. It represents the successfull realizations from a previous round, right? Considering this block out of the Restart PR context, it could be confusing why it's necessary to at that number..